lab-24-chris/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "lab-24-chris",
+  "version": "1.0.0",
+  "description": "",
+  "main": "webpack.config.js",
+  "scripts": {
+    "lint": "eslint ./",
+    "watch": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot",
+    "cover": "npm run test --karma-coverage",
+    "test": "karma start --single-run",
+    "test-watch": "karma start",
+    "build": "webpack"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ChristopherSClosser/24-ng-adventure.git"
+  },
+  "author": "",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/ChristopherSClosser/24-ng-adventure/issues"
+  },
+  "homepage": "https://github.com/ChristopherSClosser/24-ng-adventure#readme",
+  "devDependencies": {
+    "angular-mocks": "^1.6.4",
+    "jasmine": "^2.6.0",
+    "jasmine-core": "^2.6.2",
+    "karma": "^1.7.0",
+    "karma-chrome-launcher": "^2.1.1",
+    "karma-coverage": "^1.1.1",
+    "karma-istanbul": "0.0.2",
+    "karma-jasmine": "^1.1.0",
+    "karma-mocha-reporter": "^2.2.3",
+    "karma-webpack": "^2.0.3",
+    "mocha": "^3.4.2",
+    "webpack-dev-server": "^2.4.5"
+  },
+  "dependencies": {
+    "angular": "^1.6.4",
+    "babel-core": "^6.24.1",
+    "babel-loader": "^7.0.0",
+    "babel-preset-es2015": "^6.24.1",
+    "css-loader": "^0.28.3",
+    "extract-text-webpack-plugin": "^2.1.0",
+    "file-loader": "^0.11.1",
```

Comment:
this is a fun one to mess with if you want to include pictures/files into your game